src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductTaxon.xml

```diff
@@ -9,25 +9,35 @@
 
         <attribute name="normalization_context">
             <attribute name="groups">
-                <attribute>product_taxon:read</attribute>
+                <attribute>admin:product_taxon:read</attribute>
+                <attribute>shop:product_taxon:read</attribute>
             </attribute>
         </attribute>
```

Comment:
I am not saying that this blocks this PR but is this block needed now? As we also add them for each operation?

Reply:
cc / @GSadee @lchrusciel

Reply:
for operations with index shop_, like shop_get we use shop index, for admin operations we use admin, where path wasnt other than admin i just added one attribute with admin

Reply:
I understand that you put admin and shop sections and its correct. But because every endpoint has this attribute in `<collectionOperation ...` there is no need fo this attribute block here. These attributes as I remember - are usually populated for every operation, and because we have this in operation it is overridden. That is why I feel that these section (line 10-15) is not needed anymore.

Reply:
I agree that in this resource configuration, there is no need for that general `normalization_context` attribute